lib/Resources/EventDefinitions.js

```diff
@@ -61,6 +61,42 @@ export const EventDefinitions = {
 			},
 		],
 	},
+	sun_event: {
+		name: 'On Sunrise/Sunset',
+		options: [
+			{
+				id: 'type',
+				label: 'Sunrise / Sunset',
+				type: 'dropdown',
+				default: 0,
+				choices: [{id: 0, label: 'Sunrise'}, {id:1, label: 'Sunset'}],
```

Comment:
```suggestion
				choices: [{id: 'sunrise', label: 'Sunrise'}, {id:'sunset', label: 'Sunset'}],
```
I think it makes more sense to give meaningful ids to this, instead of using arbitary numbers